index.js

```diff
@@ -55,6 +55,15 @@ app.on('ready', () => {
     });
     winCount++;
     win.loadURL(url);
+    // prevent file drops to navigate the window. Instead handle explicitly
+    // file drops as string to location of file.
+    win.webContents.on('will-navigate', (event, url) => {
+      event.preventDefault();
+      const path = url.split('file://'); // if has file:// prefix...
```

Comment:
I'd recommend to use `url.parse()` here, check the `parsed.protocol === 'file:'`, and then use [`file-uri-to-path`](https://github.com/TooTallNate/file-uri-to-path) module for better Windows support.